cmd/aries-js-worker/main.go

```diff
@@ -38,13 +43,30 @@ type result struct {
 
 // All supported command handlers.
 func handlers() map[string]map[string]func(*command) *result {
-	return map[string]map[string]func(*command) *result{
-		"test": {
-			"echo":       echo,
-			"throwError": throwError,
-			"timeout":    timeout,
-		},
+	commands := getAllCommands()
+	pkgMap := make(map[string]map[string]func(*command) *result, len(commands))
+
+	for _, cmd := range commands {
+		fnMap, ok := pkgMap[cmd.Name()]
+		if !ok {
+			fnMap = make(map[string]func(*command) *result)
+		}
+
+		fnMap[cmd.Method()] = cmdExecToFn(cmd.Handle())
+		pkgMap[cmd.Name()] = fnMap
+	}
+
+	println("All commands supported are as follows")
```

Comment:
when are we printing this out?

Reply:
@sudeshrshetty  if you want to print in console use fmt

Reply:
I will remove it, right now it is just to list all available commands.

Reply:
removed it